test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -15,6 +17,24 @@ export default class PossibleIssues extends Component {
         await this.fetchIssues();
     }
 
+    getUserFeedback = async(repoName, buildName, issueName, issueCreator, accuracy) => { 
+        const feedback = await fetchData(`/api/getFeedbackUrl?repoName=${repoName}&buildName=${buildName}&issueName=${issueName}&issueCreator=${issueCreator}&accuracy=${accuracy}`);
+        let feedbackRepo, feedbackBuild, feedbackIssue, feedbackCreator, feedbackAccuracy;
+
+        if (feedback) {
+            const feedbackRes = feedback.output;
+            feedbackRepo = feedbackRes.repoName;
+            feedbackBuild = feedbackRes.buildName;
+            feedbackIssue = feedbackRes.issueName;
+            feedbackCreator = feedbackRes.issueCreator;
+            feedbackAccuracy = feedbackRes.accuracy;
+            if (accuracy === true) {
+                console.log(`The details are :-\n RepoName: ${feedbackRepo},\n buildName: ${feedbackBuild},\n issueName: ${feedbackIssue},\n creator: ${feedbackCreator}`);
+                console.log(feedbackRes);
```

Comment:
After change API return to  "error" or "success" in `feedback`, for here you can just print error or success info.